src/mds/Server.cc

```diff
@@ -117,62 +117,62 @@ void Server::create_logger()
       "Client session messages", "hcs", PerfCountersBuilder::PRIO_INTERESTING);
   plb.add_u64_counter(l_mdss_dispatch_client_request, "dispatch_client_request", "Client requests dispatched");
   plb.add_u64_counter(l_mdss_dispatch_slave_request, "dispatch_server_request", "Server requests dispatched");
-  plb.add_u64_counter(l_mdss_req_lookuphash, "req_lookuphash",
```

Comment:
Why are these counters removed?

Reply:
because i want to statistics the latency of all operations and i think the name "l_mdss_req_lookuphash_latency" is better than "l_mdss_req_lookuphash" to express it.